### PR TITLE
Changed order feeds to use new feed process

### PR DESCRIPTION
**Pull request by pcGarethJames**

### CODE REVIEW

File: Model/Feed/Type/Order/RowData.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+        $id = $order->getIncrementId();
+        $customerId = $order->getCustomerId();
+        $email = $order->getCustomerEmail();
+        $date = $order->getCreatedAt();
+
+        foreach ($order->getAllVisibleItems() as $item) {
+            $orderData[] = [
+                'OrderID' => $id,
+                'UserId' => $customerId ?: '',
```
Comment (PeterBrooksbank) [inline Model/Feed/Type/Order/RowData.php:38]: Why is user id the only one coalesced to an empty string?

**Review by PeterBrooksbank — COMMENTED** [review]
Just the one query

File: Model/Feed/Type/Order/RowData.php (hunk 1 above)
Comment (pcGarethJames) [reply Model/Feed/Type/Order/RowData.php:38]: it's the only field that's possibly not there (if it's a guest order, it'll return null), all the other fields will be there all of the time

**Review by PeterBrooksbank — APPROVED** [review]
LGTM

**Pull request merged**